docs/survey.md

```diff
@@ -1,36 +1,40 @@
 # .NET Library and Open Source Survey
 
-## When you're building class libraries, you're mostly doing this because
+## What is the primary reason why you're building class libraries?
 
-- ( ) You need to break down your own application logic
-- ( ) You need re-usable components that are used by other people's apps
-- ( ) You rarely build libraries
+- ( ) I break down my application logic
+- ( ) I build re-usable components that are used by other people's apps
+- ( ) I build them for fun
+- ( ) I rarely build libraries
 
-## When you're building libraries, you're mostly building for
+## How often do you target multiple frameworks for a single library?
 
-- ( ) a single target framework
-- ( ) multiple target frameworks
+- ( ) Very often
+- ( ) Often
+- ( ) Sometimes
+- ( ) Rarely
+- ( ) Very rarely
 
-## When you're consuming libraries, do you often see issues because different parts of your dependencies only work with specific versions of the library?
+## How often do you experience issues because different parts of your dependencies only work with specific versions of a library?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very often
+- ( ) Often
+- ( ) Sometimes
+- ( ) Rarely
+- ( ) Very rarely
 
-## Do you own NuGet packages that are available on nuget.org?
+## Do you own NuGet packages on nuget.org?
```

Comment:
What about an alternative package registry for which people need access (paid artifacts) such as @Aaronontheweb his new project?

Reply:
Tom is referring to https://sdkbin.com/ - FYI

Reply:
The goal was to distinguish private feeds (MyGet, AzDo), CI feeds (MyGet, AzDo, GH) from "official" feeds that are likely consumed by other people.